.github/workflows/release-azure-webapp-v1.yaml

```diff
@@ -0,0 +1,268 @@
+on:
+  workflow_call:
+    inputs:
+      workspace_name:
+        description: The name of the workspace to create the artifact for.
+        type: string
+        required: true
+      resource_group_name:
+        description: Web App resource group name.
+        type: string
+        required: true
+      web_app_name:
+        description: Web App name.
+        type: string
+        required: true
+      environment:
+        description: Environment where the artifact will be deployed.
+        type: string
+        required: true
+      use_staging_slot:
+        description: Use a staging slot for the deployment.
+        type: boolean
+        required: false
+        default: true
+      disable_auto_staging_deploy:
+        description: Disable automatic deployments to the staging slot.
+        type: boolean
+        required: false
+        default: false
+      use_private_agent:
+        description: Use a private agent to deploy the built artifact.
+        type: boolean
+        required: false
+        default: true
+      use_labels:
+        description: Use labels to start the right environment's GitHub runner. If use_labels is true, also use_private_agent must be set to true
+        type: boolean
+        required: false
+        default: false
+      override_labels:
+        description: Needed for special cases where the environment alone is not sufficient as a distinguishing label
+        type: string
+        required: false
+        default: ""
+
+concurrency:
+  group: ${{ github.workflow }}-cd
+  cancel-in-progress: true
+
+env:
+  BUNDLE_NAME: ${{ inputs.workspace_name }}
+  RESOURCE_GROUP_NAME: ${{ inputs.resource_group_name }}
+  WEB_APP_NAME: ${{ inputs.web_app_name }}
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    env:
+      WORKSPACE_NAME: ${{ inputs.workspace_name }}
```

Comment:
`BUNDLE_NAME` has the same value

Reply:
good catch, I wanted to remove `BUNDLE_NAME` to keep only `WORKSPACE_NAME`

Reply:
@Krusty93 I decided to keep different variable names because they have different roles despite the same content